main.rb

```diff
@@ -21,21 +32,28 @@ def director(user_answer)
   def director_add_items(user_answer)
     case user_answer
     when 7
-      # '7 Add a book'
+      add_book
     when 8
       # '8 Add a music album'
     when 9
-      # '9 Add a game'
+      puts 'Follow the prompts to add a game'
+      add_game(@all_games, @all_authors)
     when 10
```

Comment:
![image](https://user-images.githubusercontent.com/86018834/227203630-b0e3b603-292e-4081-9bc6-3369e2d7fc7d.png)

Reply:
the error occurs when the user chooses option 10 to exit from the application, please fix this error.